### DISCUSSION TITLE
tutorial-extras/SSD1306

### DISCUSSION THREAD

**Discussion by giscus** [Air-duino/document#24] [comment]
# tutorial-extras/SSD1306

简介 本章介绍使用 Air001 开发板在 Arduino 平台上点亮SSD1306屏幕的使用方法。 硬件准备 Air001开发板一块、I²C接口的OLED屏幕（SSD1306驱动）一块。 按☁️ Air001开发板入门 (/tutorial-advanced/Air001_start.html)，将Air001和DAPLink调试器使用排针排母连接。...

https://arduino.luatos.com/tutorial-extras/SSD1306.html

**Comment by djzrs** [comment]
4针的屏幕可以用吗

**Comment by YUAXI** [comment]
同样的oled显示屏同样的接线和原封不动的程序，7pin的oled点不亮，4pin的可以，不知道什么原因
